test/functional/host_collection/create_test.rb

```diff
@@ -29,6 +29,17 @@ module HammerCLIKatello
       run_cmd(%w(host-collection create --name hc1 --organization org1))
     end
 
+    it 'allows unlimited-hosts flag with no arguments' do
+      api_expects(:organizations, :index) { |par| par[:search] == "name = \"org1\"" }
+        .returns(index_response([{'id' => 1}]))
+
+      api_expects(:host_collections, :create) do |par|
+        par['unlimited_hosts'] == true
```

Comment:
```suggestion
        assert par['unlimited_hosts']
```

Reply:
Interesting.. I thought you always wanted to assert looking at some of the test .. But then I see other tests doing what you are doing.

Reply:
Ok I take back. I tested with
```
 par['unlimited_hosts'] == false
```
and it failed correctly.

Reply:
sounds like either work